### DISCUSSION TITLE
Pages should control info sidebar expansion

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#4917] [Feature Requests]
Some pages do not warrant an "auto-expanding" info sidebar... such as Docs. It just needlessly shrinks the page viewport. Ideally each page/module/route can control whether the info sidebar should have the auto-expand feature enabled.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
